Statistics.py

```diff
@@ -0,0 +1,102 @@
+# Statistics.py
+# Computes statistics and creates cost histogram
+# Ran Libeskind-Hadas, June 2020
+
+# Performs Monte Carlo simulation to compute empirical p-value and histogram
+# of solution costs.
+# https://www.ncbi.nlm.nih.gov/pmc/articles/PMC379178/
+
+from clumpr import DTLReconGraph
+import random
+import matplotlib
+matplotlib.use('tkagg') # need this so plt.show() works
```

Comment:
Please remove this `matplotlib.use('tkagg')` line as it will be addressed in #49, #52 .